src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -559,42 +547,42 @@ private void ProcessDefineHuffmanTablesMarker(int remaining)
             }
         }
 
-        private void ProcessDefineHuffmanTablesMarkerLoop(ref Huffman huffman, ref int remaining)
+        private void ProcessDefineHuffmanTablesMarkerLoop(ref HuffmanTree huffmanTree, ref int remaining)
```

Comment:
Again: Perf.
`HuffmanTree` should be a class in a "normal" C# code. I made it to be a struct because I wanted to improve data locality by removing the "array of objects" memory structure in `JpegDecoderCore.huffmanTrees`.
`HuffmanTree` has a lots of members, so copying [it inside a nested loop](https://github.com/antonfirsov/ImageSharp/blob/jpeg-optimizations/src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs#L1720) would be very slow.